### PR TITLE
3.5 release notes - Foreman Google

### PR DESCRIPTION
**Pull request by stejskalleos**

### CODE REVIEW

File: guides/doc-Release_Notes/topics/foreman.adoc [hunk 1]
```diff
@@ -7,5 +7,9 @@
 [id="foreman-upgrade-warnings"]
 == Upgrade Warnings
 
+=== Google Compute resouce as plugin
+
+Google compute resource have been extracted to the [Foreman Google](https://github.com/theforeman/foreman_google) plugin.
```
Comment (ekohl) [inline guides/doc-Release_Notes/topics/foreman.adoc:12]: Could you please describe the migration path for the user?

**Review by ekohl — CHANGES_REQUESTED** [review]

File: guides/doc-Release_Notes/topics/foreman.adoc [hunk 2]
```diff
@@ -7,5 +7,12 @@
...
```
Comment (adamlazik1) [inline guides/doc-Release_Notes/topics/foreman.adoc:12]: Link notation in Asciidoc is different from Markdown.
```suggestion
Google compute resource have been extracted to the https://github.com/theforeman/foreman_google[Foreman Google] plugin.
```

**Review by adamlazik1 — CHANGES_REQUESTED** [review]

File: guides/doc-Release_Notes/topics/foreman.adoc [hunk 3]
```diff
@@ -7,5 +7,12 @@
...
+
+Note: Foreman instances with enabled Google compute resource will be migrated automatically to the plugin.
+On new Foreman instances, Google compute resource first have to be enabled by running `foreman-installer --enable-foreman-plugin-google --enable-foreman-cli-google`
```
Comment (adamlazik1) [inline guides/doc-Release_Notes/topics/foreman.adoc:15]: Also made a wording suggestion.
```suggestion
Note: Foreman instances with enabled Google compute resource are migrated automatically to the plugin.
On new Foreman instances, you must first enable Google compute resource by running `foreman-installer --enable-foreman-plugin-google --enable-foreman-cli-google`
```

Comment (ekohl) [reply guides/doc-Release_Notes/topics/foreman.adoc:15]: I'd leave out the instructions for new instances and just stick to a note for the existing users of `foreman-gce`.

**Comment by stejskalleos** [comment]
@adamlazik1 @ekohl updated

File: guides/doc-Release_Notes/topics/foreman.adoc [hunk 4]
```diff
@@ -40,3 +40,5 @@ It is safe to remove the file or remove all content.
 
 * The `foreman_chef` plugin is unmaintained and broken since a few releases.
   It has been removed from the Installer and Packaging.
+* Google compute resource has been extracted to the https://github.com/theforeman/foreman_google[Foreman Google] plugin.
+  Note: Foreman instances with enabled Google compute resource are migrated automatically to the plugin.
```
Comment (maximiliankolb) [inline guides/doc-Release_Notes/topics/foreman.adoc:44]:
```suggestion
  Note: {ProjectServer} with enabled Google compute resource are migrated automatically to the plugin.
```

A bit unsure here. cc @Levi-Leah

File: guides/doc-Release_Notes/topics/foreman.adoc [hunk 5]
```diff
@@ -40,3 +40,5 @@ It is safe to remove the file or remove all content.
 
 * The `foreman_chef` plugin is unmaintained and broken since a few releases.
   It has been removed from the Installer and Packaging.
+* Google compute resource has been extracted to the https://github.com/theforeman/foreman_google[Foreman Google] plugin.
+  Note: {ProjectServer} with enabled Google compute resource are migrated automatically to the plugin.
```
Comment (adamlazik1) [inline guides/doc-Release_Notes/topics/foreman.adoc:44]:
```suggestion
+
NOTE: {ProjectServer}s with enabled Google compute resource are migrated automatically to the plugin.
```

**Review by adamlazik1 — COMMENTED** [review]
It might be as well made into actual note. Content does look good to me. @ekohl want to take a look?

**Review by maximiliankolb — APPROVED** [review]

**Comment by stejskalleos** [comment]
@maximiliankolb @ekohl @adamlazik1 can we merge?

**Review by adamlazik1 — APPROVED** [review]
LGTM, since @ekohl reviewed originally I would like him to have the final word.

File: guides/doc-Release_Notes/topics/foreman.adoc [hunk 6]
```diff
@@ -40,3 +40,6 @@ It is safe to remove the file or remove all content.
 
 * The `foreman_chef` plugin is unmaintained and broken since a few releases.
   It has been removed from the Installer and Packaging.
+* Google compute resource has been extracted to the https://github.com/theforeman/foreman_google[Foreman Google] plugin.
++
+NOTE: {ProjectServer}s with enabled Google compute resource are migrated automatically to the plugin.
```
Comment (ekohl) [inline guides/doc-Release_Notes/topics/foreman.adoc:45]: Language wise isn't this a better word order? I'd also merge the two sentences.
```suggestion
* Google compute resource has been extracted to the https://github.com/theforeman/foreman_google[Foreman Google] plugin. {ProjectServer}s with Google compute resources enabled are automatically migrated to the plugin.
```

**Review by ekohl — CHANGES_REQUESTED** [review]
Can you rebase as well so we get a preview of the changes?

**Review by ekohl — APPROVED** [review]
Thanks!

**Pull request merged**